cockatrice/src/decklistmodel.cpp

```diff
@@ -264,11 +264,44 @@ QModelIndex DeckListModel::findCard(const QString &cardName, const QString &zone
     return nodeToIndex(cardNode);
 }
 
-QModelIndex DeckListModel::addCard(const QString &cardName, const QString &zoneName)
+QModelIndex DeckListModel::addCard(const QString &cardName, const QString &zoneName, bool anAddAnyway)
```

Comment:
what's the `an` in `anAddAnyway`?

Maybe rename to `addIfMissingFromDb`?

Reply:
an = argument variable, n = integer... should be a b = boolean. Will fix this to be
`abAddAnyway`

Reply:
We don't use hungarian notation, please don't do that. We can tell it's an argument because it's in the argument list and we can tell it's a boolean because it's labelled `bool`.